01-getting-started/getting-started.html

```diff
@@ -1533,11 +1533,11 @@ <h4 class="author">CCDL for ALSF</h4>
 </ul>
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 <div id="about-refine.bio" class="section level2">
-<h2>About refine.bio</h2>
+<h2><span class="header-section-number">0.1</span> About refine.bio</h2>
```

Comment:
I'm just noticing that the numbering is using `0.1` etc. which is kind of ugly. Not needed to be fixed for this PR, but should probably be looked into.

Reply:
Oh. I kinda liked it like that. But yes. We can change it if we don't like.